.travis.yml

```diff
@@ -1,18 +1,17 @@
 ---
-sudo: false
+sudo: required
+services: docker
+addons:
+  apt:
+    packages:
+      - docker-ce
 language: ruby
```

Comment:
we don't need to state `language: ruby` anymore, I guess?

Reply:
yeah, i don't think this is needed anymore.